### PR TITLE
Added rosbag::Bag::isOpen

### PR DESCRIPTION
**Pull request by racko**

### CODE REVIEW

File: tools/rosbag_storage/include/rosbag/bag.h [hunk 1]
```diff
@@ -185,6 +185,8 @@ class ROSBAG_DECL Bag
 
     void swap(Bag&);
 
+    bool isOpen() const { return file_.isOpen(); }
```
Comment (dirk-thomas) [inline tools/rosbag_storage/include/rosbag/bag.h]: Please keep the implementation of this method in the cpp file even though it is short.

**Comment by racko** [comment]
Updated

**Comment by dirk-thomas** [comment]
The patch looks good. ~~Since it changes ABI it might be better to target the next ROS distro.~~ Nvm, the new method is non-virtual and should therefore not break ABI.

**Comment by dirk-thomas** [comment]
@ros-pull-request-builder retest this please

**Comment by dirk-thomas** [comment]
The patch looks good. ~~Since it changes ABI it might be better to target the next ROS distro.~~ Nvm, the new method is non-virtual and should therefore not break ABI.

**Pull request merged**